clients/java/src/main/java/io/camunda/zeebe/client/ZeebeClientBuilder.java

```diff
@@ -56,6 +57,12 @@ ZeebeClientBuilder applyEnvironmentVariableOverrides(
    */
   ZeebeClientBuilder defaultTenantId(String tenantId);
 
+  /**
+   * @param tenantIds the tenant identifiers which are used for job-activation commands when no
+   *     tenant identifiers are set. The default value is an empty list.
```

Comment:
❓ Why is the default an empty list? I think we should provide the default tenant as default similar to `defaultTenantId(String tenantId)` on line 58. What do you think?

Reply:
@korthout I set it as an empty list since the `ActivateJobsCommandImpl` doesn't reset the `tenantIds` list when a new `tenantId` is added. If we provide the `<default>` tenant ID in the property, users will be unable to remove it, so they will always include it in their `tenantIds` list.

Alternatively, we can do the following in `ActivateJobsCommandImpl` :
* Method `tenantIds(List)` resets the `tenantIds` list
* Method `tenantIds(String...)` resets the `tenantIds` list
* Method `tenantId(Stromg)` doesn't reset the `tenantIds` list since it can be called multiple times

WDYT?

Reply:
Ah yes, I guess it's difficult to add this behavior. 

🔧 If we want to make the `List` and `String...` methods reset the tenantIds list than we should document this in `CommandWithOneOrMoreTenantsStep`.

💡 For `tenantId()` we can build it in one of two ways:

1. Keep track of added tenantIds in a `Set<String>` field of `ActivateJobsCommandImpl`
   - Instantiate the Set with the configured default tenantIds
   - Add to the Set when calling `tenantId(String)`
   - Use this Set in the [send method](https://github.com/camunda/zeebe/blob/92da6bb2b54338055414ed47e519a797bb780427/clients/java/src/main/java/io/camunda/zeebe/client/impl/command/ActivateJobsCommandImpl.java#L111) to add the tenants from the set to the request builder before building the request
   - Automatically removes duplicates
2. Keep track of whether `tenantId` or `tenantIds` have been called or not
   - Add a boolean field to `ActivateJobsCommandImpl`, starting as `false`
   - When `tenantId(String)`, `tenantIds(List)`, or `tenantIds(String...)` is called, set the flag to `true`
   - In the [send method](https://github.com/camunda/zeebe/blob/92da6bb2b54338055414ed47e519a797bb780427/clients/java/src/main/java/io/camunda/zeebe/client/impl/command/ActivateJobsCommandImpl.java#L111) set the default tenants on the request builder only if the flag is `false`

Reply:
Or (similar to 2):
3. Keep track of whether `tenantId` or `tenantIds` have been called or not
    * Add a boolean field to ActivateJobsCommandImpl, starting as `true`
    * When `tenantId(String)`, `tenantIds(List)`, or `tenantIds(String...)` are called
       * reset the `tenantIds` list
       * set the flag to `false` 
    * For any `tenantId(String)`, `tenantIds(List)`, or `tenantIds(String...)` call, check if the `tenantId` is already present in the `tenantIds` list before adding it. This avoids using a `Set` which complicates things since we would need to reset the list every time to avoid adding duplicates.

Reply:
After a discussion, we'll go with [1] where we'll keep separate `Set`s for default and user-defined tenants, which will be set on the `builder` instance when the `send` command is called.